features/ink-enhancement.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-09-09
-name: Ink API
-description: TODO
+name: Ink enhancement
```

Comment:
It's not clear to me where this is coming from. On first look, "Ink (API)" seems to be much more common.

(Though it's possible I'm looking in the wrong places—I'd be happy to be proven wrong with examples of developers referring to this as "ink enhancement".)

```suggestion
name: Ink
```

Reply:
The word enhancement comes from the fact that this API was born out of the need for a faster way to draw ink trails on a canvas. The usual way involves listening for pointer events and drawing shapes on the canvas as a result of them. However, the time it takes for the pointer events to be dispatched to and handled by the webapp is sometimes noticeable to users when drawing quickly. It's visible in the form of your drawing lagging slightly behind your pencil or finger, which can feel weird.
The Ink API solves this by allowing you to bypass the entire process and plug directly into the compositor. You still draw your shapes as before, but now you have a way to tell the device "please draw faster, as the pointer moves, and then I'll handle the events as they get dispatched".

In that sense, it _is_ an enhancement. But it's not an enhancement of an existing _API_. I agree with calling this just **Ink**.